rllib/examples/offline_rl.py

```diff
@@ -119,7 +119,9 @@
     # Example on how to do evaluation on the trained Trainer
     # using the data from our buffer.
     # Get a sample (MultiAgentBatch -> SampleBatch).
-    batch = replay_buffer.replay().policy_batches["default_policy"]
+    batch = replay_buffer.sample(num_items=config["train_batch_size"]).policy_batches[
```

Comment:
Nit: Can we split this into two separate statements. It's very confusing, merged like this.